app/services/atmosphere/flavor_updater.rb

```diff
@@ -0,0 +1,56 @@
+# Populate the Atmosphere database with information regarding the
+# available virtual machine flavors for each compute site.
+module Atmosphere
+  class FlavorUpdater
+    def initialize(tenant)
+      @tenant = tenant
+    end
+
+    def execute
+      update_existing_flavors
+      purge_non_existing_flavors
+    rescue Exception => e
```

Comment:
Avoid rescuing the `Exception` class. Perhaps you meant to rescue `StandardError`?